apps/api/src/database/migrations/1714646233635-migration.ts

```diff
@@ -9,7 +9,7 @@ export class Migration1714646233635 implements MigrationInterface {
                 name: 'jobs',
                 columns: [
                     {
-                        name: 'job_id',
+                        name: 'id',
```

Comment:
Why was this changes to id from job_id? You could have mapped the required column using name attribute in Entity?

Reply:
Okay, reverting the changes.

Reply:
Updated migrations file